tests/files/env_config.yml

```diff
@@ -0,0 +1,9 @@
+environments:
+  development:
+    prefix: Dev-Prefix-
+
+  staging:
+    prefix: Stage-Prefix-
+
+  production:
+    prefix: Prod-Prefix-
```

Comment:
I'm on board with your suggestion, but I'm treating it orthogonally to environments. My first step here is simply decoupling regions from environments.